org.opent2t.sample.binaryswitch.superpopular/com.insteon.binaryswitch/js/manifest.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest>
+  <!-- associated voice handler -->
+  <voice id="org.opent2t.sample.lightcontrol.cortana.vcd.superpopular" />
+  <!-- associated schemas -->
```

Comment:
Can you add a quick note in teh description on how you tested this? Manual testing, unit tests etc ? thanks!